test/integration/stats_integration_test.cc

```diff
@@ -286,8 +288,8 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithFakeSymbolTable) {
   // If you encounter a failure here, please see
   // https://github.com/envoyproxy/envoy/blob/master/source/docs/stats.md#stats-memory-tests
   // for details on how to fix.
-  EXPECT_MEMORY_EQ(m_per_cluster, 44491);
-  EXPECT_MEMORY_LE(m_per_cluster, 44811);
+  EXPECT_MEMORY_EQ(m_per_cluster, 47707);
+  EXPECT_MEMORY_LE(m_per_cluster, 47771);
```

Comment:
can you round this up  47771 to 48000? IDK how these exact values got derived but really these _LE values are by nature approximate as they measure byte-count consumption across platforms and compile-modes.

We're just having the _LE versions to help catch major changes during local tests, without waiting for the CI "release" build where the _EQ version is tested.

Reply:
Thanks, will update the values.